### PR TITLE
Add custom editors to work with git

### PR DESCRIPTION
**Pull request by adityasiwan**
Add your own editors like sublime etc. to work with git.

### CODE REVIEW

**Comment by hemanth** [comment]
`[doxie] Invalid JSON input: “Unexpected token $”.`

**Comment by adityasiwan** [comment]
Removed $ which was giving error.

File: tips.json [hunk 1]
```diff
@@ -254,6 +254,9 @@
 }, {
     "title": "Make git case sensitive.",
     "tip": "git config --global core.ignorecase false"
+},{
+    "title": "Add custom editors.",
```
Comment (Tossaporns) [inline tips.json]: H

File: tips.json [hunk 2]
```diff
@@ -254,6 +254,9 @@
...
+    "tip": "git config core.editor "MY_EDITOR""
```
Comment (Tossaporns) [inline tips.json]: H

File: tips.json (hunk 1 above)
Comment (Tossaporns) [inline tips.json]: G

File: tips.json (hunk 2 above)
Comment (Tossaporns) [inline tips.json]: Y

**Comment by hemanth** [comment]
@adityasiwan Did you try: `"tip": "git config core.editor '$MY_EDITOR'"` ?

**Comment by adityasiwan** [comment]
Yes here is the one i tried and is working fine.

```git config --global core.editor "'c:/program files/sublime text 2/sublime_text.exe' -w"```

File: tips.json (hunk 2 above)
Comment (hemanth) [inline tips.json]: This should be `"tip": "git config core.editor '$EDITOR'"` basically it must be a valid JSON.

**Comment by adityasiwan** [comment]
I will reopen this once I update my fork and avoid merge conflicts.

**Comment by denisbalyko** [comment]
@adityasiwan you could do 'git rebase master'.

**Pull request closed without merging**